test/helpers/minitest_helper.rb

```diff
@@ -50,11 +52,13 @@
 if ENV['RM_INFO'] || ENV['TEAMCITY_RAKE_RUNNER_MODE'] # RubyMine
   puts "Running tests from RubyMine, using RubyMine test reporter."
   ENV.delete('RM_INFO') # Delete this environment variable because it forces use of only RubyMineReporter
-  Minitest::Reporters.use! [Minitest::Reporters::RubyMineReporter.new]
+  # Minitest::Reporters.use! [Minitest::Reporters::RubyMineReporter.new]
+  Minitest::Reporters.use! [Minitest::Reporters::RubyMineReporter.new, Minitest::Reporters::JUnitReporter.new(reports_dir="test/reports", empty=false)] 
```

Comment:
Do we want to merge these changes (same as line 62-63)? If so, we should probably check with Matt since this will end-up being rolled-up into the master branch.

Reply:
done. commented local testing lines.